litgpt/pretrain.py

```diff
@@ -244,15 +243,14 @@ def fit(
                 samples=(state["iter_num"] * train.micro_batch_size),
                 lengths=(state["iter_num"] * train.micro_batch_size * model.max_seq_length),
             )
+            remaining_days = (t1 - total_t0) / (state["iter_num"] - initial_iter) * (max_iters - state["iter_num"]) / 1440
```

Comment:
Isnt this seconds to days?

```suggestion
            remaining_days = (t1 - total_t0) / (state["iter_num"] - initial_iter) * (max_iters - state["iter_num"]) / 86400
```